### ISSUE TITLE
[Due_Date]진행 상황 보고 (지영)

### ISSUE THREAD

**Issue by yesjiyoung** [HyeseonLee/OSS_Hola#95]
++로그인/ 회원가입 페이지 점검
- [x] 로그인 페이지 관련 html / css 수정 완료, back 연결 완료
- [x]  회원가입 페이지 관련 html / css 수정 완료, back 연결 완료
- [x] 관련 error page html / css 수정 완료, back 연결 완료

++ 위의 작업 완료 후 to do it 리스트 추가예정
- [x] 스쿼트 모델 훈련 다시 하기
- [x] 카카오 API 음성 추출하기
- [x] aidetail page css path 문제해결 

++ 전체적인 디자인 점검
- [x] detail.html 수정 + 완료
- [x] aiHome.html 수정 + 완료
- [x] 데이터 다 받은 후, popVideo.html 코드 수정
- [x] aidetail.html 디자인 시작 

++ 배포
- [ ] 10시 반 헤로쿠 배포 시작 

++ 오류
- [ ] 로그인 안했을 경우, 댓글 창 submit버튼 안먹히게 하고 로그인 페이지로 돌리기

**Comment by yesjiyoung** [comment]
- 회원가입 
signup.html

![image](https://user-images.githubusercontent.com/51046701/85653274-d1190580-b6e7-11ea-813a-6a2c4a1f182a.png)

signup_correct.html
![image](https://user-images.githubusercontent.com/51046701/85653500-176e6480-b6e8-11ea-8c56-0cabaa3a3816.png)

signup_uncorrect.html
![image](https://user-images.githubusercontent.com/51046701/85653568-2f45e880-b6e8-11ea-8eba-e53ead2debf0.png)


- 로그인
login.html
![image](https://user-images.githubusercontent.com/51046701/85653645-48e73000-b6e8-11ea-802d-4586e9a3be0a.png)

unlog.html
![image](https://user-images.githubusercontent.com/51046701/85653712-5bfa0000-b6e8-11ea-93ac-0d9f9f6daa32.png)

**Comment by yesjiyoung** [comment]
**[Back-end] 기능 수정 안내드립니다.**

**회원가입**
-장고에서 지원하는 user모델의 필드가 부족하다는 생각이 들어, abstractuser 를 import하여 저희가 원하는 방식으로 custom 하였습니다. ( + nickname, age, workingout )
-모델은 새로운 필드와 함께 완벽하게 구현되었으나, 사용자가 입력하는 input과정에서 radio값을 어떻게 받아와야 할 지.. 그 부분이 해결이 되지 않아 Back-end 회의결과 radio형태로 구현된 age와 workingout을 빼는쪽으로 결정을 하였습니다.
-nickname은 radio가 아닌 textfield이기 때문에 사용자 입력과정에 문제가 없으나, 댓글을 쓸 때 username으로 간다면 nickname의 필요성을 느끼지 못하여 nickname 필드를 빼기로 결정하였습니다. ( 사실 저는 닉네임으로 구현하고 싶기도 해서.. 이부분은 추후 시간이나면 수정하도록 하겠습니다!)

**[Front-end] html 수정 안내드립니다.**

**회원가입, 로그인**
위의 [Back-end]의 기능 수정과 관련되어 html이 상당부분 변경된 점 양해부탁드립니다.. ㅠ

@Jinnan-Yun  @YuJigun 담당, 진난님 지건님 피드백부탁드리겠습니다!

**Comment by Jinnan-Yun** [comment]
네 확인했습니다! 지영님 혹시 제가 방금 서버를 켰는데 회원가입 창이 위 사진처럼 뜨지 않는데ㅠㅠ 혹시 왜 그러는지 아실까요..?

**Comment by yesjiyoung** [comment]
> 네 확인했습니다! 지영님 혹시 제가 방금 서버를 켰는데 회원가입 창이 위 사진처럼 뜨지 않는데ㅠㅠ 혹시 왜 그러는지 아실까요..?

넵 잠시만요! 확인해보겠습니다!

**Comment by yesjiyoung** [comment]
@Jinnan-Yun  혹시 pull 하셨나요!? 저한테는 정상적으로 보입니다.ㅜㅜ

**Comment by Jinnan-Yun** [comment]
앗 저도 이제 작동합니다!! 
혹시 회원가입 창, 로그인 창 상단에 signup, login 박스가 존재하는데 하단에 큰 박스에도 '회원가입', '로그인'이라고 내용이 나와 있으니 둘 중 하나만 있으면 어떨까요? 조금 더 깔끔해보이지 않을까 해서요..!!

**Comment by yesjiyoung** [comment]
@Jinnan-Yun  넵! 진난님이 직접 수정 해주셔도 좋습니다 :)!!ㅎㅎ 피드백 감사합니다~

**Comment by Jinnan-Yun** [comment]
@yesjiyoung 넵  알겠습니다 👍

**Comment by yesjiyoung** [comment]
detail.html
 
![image](https://user-images.githubusercontent.com/51046701/85728665-16165980-b733-11ea-8034-28fd042bf86d.png)

**Comment by yesjiyoung** [comment]
popVidoe.html
![image](https://user-images.githubusercontent.com/51046701/85729753-08ad9f00-b734-11ea-99bb-7f7a5b00af7a.png)
